lib/util/profiles/profile_widget.dart

```diff
@@ -20,6 +21,7 @@ class ProfileWidget extends StatelessWidget {
     this.showDescription = false,
     this.actionWidget,
     this.isTappable = true,
+    this.customAction,
     this.onPop,
```

Comment:
1) customAction should be called "onTap"
2) please make it a Function() instead of VoidCallback
3) there should be an assertion that onTap and onPop cannot be set at the same time